src/main/java/com/google/codeu/data/Datastore.java

```diff
@@ -47,6 +47,8 @@ public void storeMessage(Message message) {
     messageEntity.setProperty("text", message.getText());
     messageEntity.setProperty("timestamp", message.getTimestamp());
     messageEntity.setProperty("recipient", message.getRecipient());
+    messageEntity.setProperty(
+        "sentimentScore", message.getSentimentScore()); // stores the sentimentScore
```

Comment:
Remove the inline comment //stores the sentimentScore. It is inferred from the code that we are storing it in the messageEntity.